src/app/page.ts

```diff
@@ -0,0 +1,20 @@
+import { main } from '@components/tags';
+import { movieService } from '@services/movie.service';
+
+import { BaseComponent } from './components/base-component';
+import { Header } from './components/header/header';
+import { MovieListPage } from './pages/movie-list';
+
+class PageWrapperComponent extends BaseComponent {
+  constructor() {
+    super(
+      {
+        className: 'page-wrapper',
+      },
+      Header.bind(null)(),
```

Comment:
Header.bind(null)() seems unnecessary. If Header is a component function, calling Header() directly should be enough
```suggestion
      Header()
```